### DISCUSSION TITLE
is this database still updated?

### DISCUSSION THREAD

**Discussion by malwaregarry** [stoneatom/stonedb#1983] [General]
Hi, I am from the SQLancer team.
The current implementation of StoneDB in SQLancer is broken and we are considering removing support for StoneDB if  it is no longer maintained.

**Comment by bhzhu203** [comment]
Yes ,The  latest updates are all about Mysql 8.0 (maybe 5.7 is abandoned)  .

https://www.stoneatom.com/download
https://www.stoneatom.com/StonedbDocs?id=167

**Comment by malwaregarry** [comment]
I can't seem to access the links. Only stonedb.io is accessible to me

Are the updates on another repository?

**Reply by bhzhu203** [reply]
> I can't seem to access the links. Only stonedb.io is accessible to me
> 
> Are the updates on another repository?

Maybe it is the close source  . I have not seen the latest sources of the latest release .

here is the latest binary:
https://stoneatom-static.oss-cn-hangzhou.aliyuncs.com/stonedb-ee-8.0-v2.7.1.el7.x86_64.tar.gz

**Comment by wangboo** [comment]
This project is dead.
